### DISCUSSION TITLE
How to cause chapter markers to show up in progress bar

### DISCUSSION THREAD

**Discussion by legocabin** [videojs/video.js#8202] [Q&A]
I've implemented chapters in my videojs player using a VTT file

```
var track = {
        kind: 'chapters',
        src: chapterurl,
        default: '',
      };
```
and then calling  `player.addRemoteTextTrack(track);` after I've initialized the player. 

This has caused the Chapters menu to appear in the control bar which is great. 
My original assumptions was that this would also cause chapter markers to show up in the progress bar also. 

so questions would be:
1. Should it cause chapter markers to show up automatically?
2. If not, what should I do to cause it show up? (is this something videojs supports etc...)

Thanks!

**Comment by gkatsev** [comment]
The progress bar chapter markers isn't something that Video.js supports out of the box. Unfortunately, I'm not sure if there's an up-to-date plugin that handles this.

**Reply by legocabin** [reply]
Got it thanks! Does it support progress bar markers in general? (Could I for example input markers via the same VTT file, but not through adding a chapters track)

**Reply by gkatsev** [reply]
Unfortunately, there isn't such functionality. There's this old plugin https://github.com/spchuang/videojs-markers that you could use as a starting point for your functionality.
